packages/core/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     // basic stuff, or everything breaks
     "module": "ESNext",
-    "moduleResolution": "node16",
+    "moduleResolution": "node",
```

Comment:
See earlier comments about putting the `bindings` in `packages/wasm` instead of `core`.